docs/DesignMDIngestion.md

```diff
@@ -0,0 +1,61 @@
+# Generic Metadata Ingestion to MongoDB
+
+## Description
+
+This is the design document discussing the ingestion process of existing metadata
+into Zenko with MongoDB.
+
+The primary use case is to ingest existing metadata from the RING and S3 Connector;
```

Comment:
Ingesting existing metadata is not a use case, it's the realization of the use case. Why would someone ingest existing from metadata? (apply to the whole paragraph)